twig/BS3/fields/selectMultipleInput.twig

```diff
@@ -23,13 +23,13 @@
             {% endfor %}
         </select>
         <span class="input-group-btn">
-            <button class="btn btn-success tool_button" id="{{ id }}-all" type="button" title="Select All">
-                <i class="fa fa-check"></i>
+            <button class="btn btn-success tool_button" id="{{ id }}-all" type="button">
+                <i class="fa fa-check"></i><span class="sr-only">Select All</span>
             </button>
         </span>
         <span class="input-group-btn">
-            <button class="btn btn-danger tool_button" id="{{ id }}-none" type="button" title="Remove All">
-                <i class="fa fa-times"></i>
+            <button class="btn btn-danger tool_button" id="{{ id }}-none" type="button">
+                <i class="fa fa-times"></i><span class="sr-only">Select All</span>
```

Comment:
We want the title on this so people can hover over and know what it does.  This control isn't just for screen readers.

Reply:
Got it!